core/src/components/toast/toast.ionic.scss

```diff
@@ -0,0 +1,76 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@use "./toast.common";
+
+// Ionic Toast
+// --------------------------------------------------
+
+:host {
+  --background: #{globals.$ionic-color-neutral-1200};
+  --border-radius: #{globals.$ionic-border-radius-400};
+  --box-shadow: #{globals.$ionic-elevation-400};
+  --button-color: #{globals.$ionic-color-base-white};
+  --color: #{globals.$ionic-color-base-white};
+  --max-width: 343px;
+  --start: 8px;
+  --end: 8px;
+
+  @include globals.typography(globals.$ionic-body-md-medium);
+
+  // TODO(ROU-10853): replace this value with a layer token.
+  z-index: 1001;
+}
+
+.toast-wrapper {
+  @include globals.margin(auto);
+
+  display: block;
+  position: absolute;
+
+  opacity: 0.01;
+
+  // TODO(ROU-10853): replace this value with a layer token.
+  z-index: 10;
+}
+
+.toast-content {
+  @include globals.padding(globals.$ionic-space-300, globals.$ionic-space-400);
+}
+
+// Toast Button
+// --------------------------------------------------
+
+.toast-layout-baseline .toast-button-group-start {
+  @include globals.margin(null, null, null, globals.$ionic-space-400);
```

Comment:
Should we do `gap` here instead?

Reply:
No, if we replace margin with gap then there wouldn't be any spacing at the beginning. Here's an example:

![Screenshot 2024-10-03 at 4 42 32 PM](https://github.com/user-attachments/assets/ca370d58-7430-4c76-a1de-ee5f8979fc78)

This is because gap only adds spacing between and not at the ends. I did realize that gap should go within the button groups though: https://github.com/ionic-team/ionic-framework/pull/29922/commits/f0bc6f0b53c4dc4bdcaf085c18cf3161d4c6a70f

Reply:
Ahh yeah sorry I commented on the wrong style! I meant the gap between the button and the text.

Reply:
lol, you were just testing me